examples/config/v0.5.3-selinux/go-app-counter/kustomization.yaml

```
@@ -0,0 +1,20 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+# Patch the bytecode.yaml to change tag on the "url" field (which is an
+# image) to new value. This actually overwrites the image with the same value.
+patches:
+  - target:
+      kind: BpfApplication
+      name: app-counter
+    patch: |-
+      - op: replace
+        path: "/spec/bytecode/image/url"
+        value: quay.io/bpfman-bytecode/go-app-counter:vv0.5.3
```

```suggestion
        value: quay.io/bpfman-bytecode/go-app-counter:v0.5.3
```
Global change, this needs to be changed everywhere.

---

Done.